administration/src/project-configs/koblenz/dataPrivacyBase.tsx

```diff
@@ -39,25 +40,24 @@ r.substr(0, r.search("<p")) + r.substr(r.search("</p>") + 4)
 Copy the resulting string into the htmltojsx converter.
 */
 
-// TODO 1548 Provide data privacy text
 export const DataPrivacyBaseText = (): ReactElement => (
   <div>
+    <b>Vorbemerkungen:</b>
     <p style={{ marginTop: '5pt', marginBottom: '5pt' }}>
       Diese Datenschutzerklärung bezieht sich auf die Verarbeitung personenbezogener Daten im Rahmen des digitalen
-      NürnbergPass einschließlich verbundener Dienste.
+      KoblenzPass einschließlich verbundener Dienste.
```

Comment:
See above, here it should probably be also `KobelnzPasses` then, right? Or otherwise also `KoblenzPass` above.